pkg/controller/metrics/metrics.go

```diff
@@ -97,8 +113,35 @@ func (mc *Calculator) Start(stopCh <-chan struct{}) error {
 				installedTotal = installedTotal + 1
 			}
 		}
+		// jobs metrics
+		jobs := &batchv1.JobList{}
+		err = mc.Client.List(context.Background(), &client.ListOptions{}, jobs)
```

Comment:
This is listing every job in the cluster, lets use the labels in the query so we *only* fetch install/uninstall jobs and save a lot of memory.

Reply:
This should be possible with some params to ListOptions.